libkbfs/config_local.go

```diff
@@ -776,6 +776,9 @@ func (c *ConfigLocal) Shutdown() error {
 				continue
 			}
 			for _, fbo := range kbfsOps.ops {
+				// Use a fresh context so we don't
```

Comment:
If this is a possibility, then why can we get away with passing `ctx` into `KBFSOps.Shutdown()` below?  If the passed-in `ctx` is cancelled, doesn't it indicate a real error (rather than an on-purpose testing error) anyway?